### DISCUSSION TITLE
Aug 31 meeting agenda

### DISCUSSION THREAD

**Discussion by xcorail** [ossf/wg-best-practices-os-developers#16] [General] [answered]
👋🏾 folks!
For this meeting I expect many newcomers, so I propose the following agenda. Please update / comment ...

- Round table, introductions: 10'
- Overview of the working group projects: 10' @xcorail 
- Details of the Learning platform project: @blabla1337 are you up to do it? Can you add a timing?
- the [CII Best Practices badge](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/cii-bp-badge-intro.pptx) - @david-a-wheeler can you add a timing?
- the "developing secure software" course that I've drafted, [as noted here](https://github.com/ossf/wg-best-practices-os-developers/issues/11) - @david-a-wheeler can you add a timing?

See you there!

**Comment by david-a-wheeler** [comment]
I suggest 10 minutes for the CII best practices badge presentation, and 10 minutes for the "developing secure software" course I drafted. But it's your meeting, so you decide!

**Comment by blabla1337** [comment, marked as answer]
Yeah I'm up for it I think I need around 20 min max

**Comment by david-a-wheeler** [comment]
FYI - here are some links where you can see more about the CII Best Practices:

* [Criteria for *just* the passing level](https://bestpractices.coreinfrastructure.org/criteria)
* [All criteria (passing, silver, and gold levels)](https://bestpractices.coreinfrastructure.org/criteria)
* [All criteria (passing, silver, and gold levels) *with* details & rationale](https://bestpractices.coreinfrastructure.org/en/criteria?details=true&rationale=true)
* [Discussion about the CII criteria](https://bestpractices.coreinfrastructure.org/criteria_discussion)
* [Criteria statistics](https://bestpractices.coreinfrastructure.org/criteria_stats)
* [Project statistics](https://bestpractices.coreinfrastructure.org/project_stats)

Overall:
* [CII Best Practices badge website](https://bestpractices.coreinfrastructure.org)
* [CII Best Practices badge development repo (on GitHub)](https://github.com/coreinfrastructure/best-practices-badge), code is under MIT license

**Reply by bkimminich** [reply]
🥇 https://bestpractices.coreinfrastructure.org/de/projects/223 ... not bragging or anything, but... 🤡
